.github/utils/ugbio_members_matrix.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
+import os
+import json
+import argparse
+
+INTERNAL_MEMBERS = ["core", "omics"]
+
+def build_members_matrix(src_folder: str, keep_internal: bool):
+    # List directories in the src_folder that don't contain "__"
+    directories = [
+        d for d in os.listdir(src_folder)
+        if os.path.isdir(os.path.join(src_folder, d)) and "__" not in d
+    ]
+
+    if not keep_internal:
+        directories = [d for d in directories if d not in INTERNAL_MEMBERS]
+
+    print(json.dumps(directories, indent=4, sort_keys=True))
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--src-folder", required=False, help="Path to src folder", default="src")
+    parser.add_argument("--keep-internal", help="Keep internal ugbio members", action="store_true")
```

Comment:
Instead of hardcoding the internal packages and exposing the `--keep-internal` I would make the code more agnostic with a `--exclude` parameter.
Something like:
`parser.add_argument("--exclude", help="Exclude folders (space separated)", nargs="+")`

Reply:
Thanks for this suggestion, but our motivation was to store the internal members in one place instead of listing them in every action that needs them. (there will be more I assume). I think this keeps it simplest.